adc/ad7779/ad7779.h

```diff
@@ -89,4 +89,58 @@ int ad7779_get_status(uint8_t *status_buf);
 /* For debugging purposes */
 int ad7779_print_status(void);
 
+/* Logging */
+#define COL_RED    "\033[1;31m"
+#define COL_CYAN   "\033[1;36m"
+#define COL_NORMAL "\033[0m"
+
+#define LOG_TAG "ad7779: "
+
+#ifdef NDEBUG
+#define log_debug(fmt, ...)
+#else
+#define log_debug(fmt, ...) \
+	do { \
+		printf(LOG_TAG fmt "\n", ##__VA_ARGS__); \
+	} while (0)
+#endif
+
+#define log_info(fmt, ...) \
+	do { \
+		printf(LOG_TAG COL_CYAN fmt COL_NORMAL "\n", ##__VA_ARGS__); \
+	} while (0)
+#define log_error(fmt, ...) \
+	do { \
+		printf(LOG_TAG COL_RED fmt COL_NORMAL "\n", ##__VA_ARGS__); \
+	} while (0)
+
+/* DMA api */
```

Comment:
maybe we should write a comment here that it's an internal low-layer API to be implemented when porting the driver to new target?

Or maybe the internal api should be in separate header? (the above functions are external API or what?)

Reply:
Moved to `ad7779-low.h` + additional comment